server/data/products.js

```diff
@@ -166,11 +169,13 @@ let exportedMethods = {
     return soredTagByLikes;
   },
   async updateCount(prodId, liked) {
-    if(typeof prodId === "undefined") throw"prodId is not provided";
-    if(typeof liked === "undefined") throw "liked is not provided";
-    if(typeof prodId != "string") throw"prodId is not a string";
-    if(typeof liked != "boolean") throw"liked is not a boolean value";
-    if(prodId.trim() === "") throw"prodId is blank";
+    //This was used to populate using seed file
+
+    // if (typeof prodId === "undefined") throw "prodId is not provided";
+    // if (typeof liked === "undefined") throw "liked is not provided";
+    // if (typeof prodId != "string") throw "prodId is not a string";
+    // if (typeof liked != "boolean") throw "liked is not a boolean value";
+    // if (prodId.trim() === "") throw "prodId is blank";
```

Comment:
Dont we need this, if need please remove this

Reply:
Oishik is passing a few values while in the seedfile which he's checking. This was causing the error when accessing reviews and products